ArduPlane/events.cpp

```diff
@@ -169,9 +169,13 @@ void Plane::handle_battery_failsafe(const char *type_str, const int8_t action)
             break;
 
         case Failsafe_Action_Terminate:
+#if ADVANCED_FAILSAFE == ENABLED
             char battery_type_str[17];
             snprintf(battery_type_str, 17, "%s battery", type_str);
             afs.gcs_terminate(true, battery_type_str);
+#else
+            disarm_motors();
```

Comment:
This is wrong IMO. Disarming motors is quite different then terminating the vehicle. If AFS isn't available you shouldn't be able to terminate. (Although arguably we have a preexisting bug as we didn't respect the return value on `gcs_terminate` anyways)

Reply:
Then we also have a different bug in the other vehicles because this matches their behavior

Reply:
On other vehicles disarming motors is actually symmetric with terminating. On plane you will continue to glide away gently into the ground, which can be a very very large distance that the vehicle travels. (That being said a disarm only failsafe may be desirable). Terminate should minimize any and all movement. On a copter stopping motors is termination :)

Reply:
So... How about we reverse all flight surfaces except throttle?